project/plugins.sbt

```diff
@@ -2,4 +2,4 @@ addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.11")
 
 addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.2")
 
-addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "2.6")
+addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "3.7")
```

Comment:
This commit also updates the `sbt-sonatype` plugin - apparently this version is much faster to upload to Sonatype!

* https://users.scala-lang.org/t/10x-faster-release-with-sbt-sonatype-3-6/5046
* https://twitter.com/taroleo/status/1169777599360331776

Reply:
The configuration wasn't correct for this stuff when I merged this commit, but I eventually got it straightened out on master!

I think bd533dbf3 & 47d01fb86 were the crucial commits.